src/System.Net.Security/tests/UnitTests/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicyTest.cs

```diff
@@ -26,6 +26,14 @@ public void Constructor_ServiceNameCollection_ZeroElementsParam()
             Assert.Equal(typeof(ArgumentException), ex.GetType());
         }
 
+        [Fact]
+        public void Constructor_PolicyEnforcementChannelBinding_NeverParam()
+        {
+            var customChannelBinding = new MockCustomChannelBinding();
+            var ex = AssertExtensions.Throws<ArgumentException>("policyEnforcement", () => new ExtendedProtectionPolicy(PolicyEnforcement.Never, customChannelBinding));
+            Assert.Equal(typeof(ArgumentException), ex.GetType());
```

Comment:
What is this type check for? You are already checking with Throws that the exception is of type ArgumentException